packages/react-ui-core/src/index.js

```diff
@@ -48,6 +48,7 @@ export {
 
 export {
   List,
+  HighlightedList,
```

Comment:
This is not a component anymore is it?

Reply:
Whoops. Sorry. I'll remove that

Reply:
Yeah, we need to be careful here. I think this would actually break things outside of this repo.

Reply:
Gotcha. yeah, I'll be more careful with that going forward. That was my bad